contrib/broker/k8s/Dockerfile

```diff
@@ -20,13 +20,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && apt-get update \
  && apt-get install -y \
     ca-certificates \
+    curl \
+ && curl -Ls -o /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+ && mkdir -p /opt/services \
+ && tar -C /opt/services -x -z -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    --strip-components=1 linux-amd64/helm \
+ # cleanup
```

Comment:
If any one piece of that fails, the whole RUN directive fails and the Docker build fails in turn, which is what we'd want.